tests/rbd/rbd_snap_clone_imported_image.py

```diff
@@ -58,9 +60,20 @@ def test_snap_clone(rbd, pool_type, **kw):
     finally:
         if not kw.get("config").get("do_not_cleanup_pool"):
             rbd.clean_up(pools=[pool])
+            config = kw["config"]
+            cephadm = CephAdmin(cluster=ceph_cluster, **config)
+            rados_obj = RadosOrchestrator(node=cephadm)
+            list_of_pools = rados_obj.list_pools()
```

Comment:
Instead of adding this logic in each test can we add this logic in `rbd.clean_up` script itself,
so that it will be applicable for all other tests which leaves the pool at the end.

Reply:
Added logic in rbd.clean_up script. Have put it under a flag to make sure other scripts do not regress which are intentionally just cleaning up specific pools.